scripts/backend_test_shards.json

```diff
@@ -44,6 +44,7 @@
         "core.controllers.topics_and_skills_dashboard_test",
         "extensions.value_generators.models.generators_test",
         "core.controllers.admin_test",
+        "core.controllers.contributor_dashboard_admin_test",
```

Comment:
Can this be moved to shard 5 (for load balancing)?

Reply:
Done!